style.css

```
@@ -572,19 +562,11 @@ body[data-bg="wallpaper"] .tooltip-text {
-	display: block;
-	/* Ensure single-word labels remain visible */
-}
-
-.icon-item:hover .label.short.one {
-	display: none;
-	/* Ensure single-word labels remain visible */
+/* Expand label on hover to show full name */
+.icon-item:hover .label {
+	white-space: normal;
+	overflow: visible;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)